stripe/resource.py

```diff
@@ -106,6 +106,89 @@ def _serialize_list(array, previous):
     return params
 
 
+def nested_resource_class_methods(resource, path=None, operations=None):
+    if path is None:
+        path = "%ss" % resource
+    if operations is None:
+        operations = ['create', 'retrieve', 'update', 'delete', 'list']
```

Comment:
@ob-stripe Relating to https://github.com/stripe/stripe-ruby/pull/600, mind if we require an explicit list of supported operations here?